internal/locate/region_request.go

```diff
@@ -1158,6 +1265,7 @@ func (s *RegionRequestSender) sendReqToRegion(bo *retry.Backoffer, rpcCtx *RPCCo
 	if e := tikvrpc.SetContext(req, rpcCtx.Meta, rpcCtx.Peer); e != nil {
 		return nil, false, err
 	}
+	rpcCtx.overrides.patchContext(&req.Context)
```

Comment:
Just my personal preference:
If this is written as something like `rpcCtx.contextPatcher.applyTo(&req.Context)` then it would be easier to understand without checking out what the function does.

Reply:
OK, I change the type name.